seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/SeataAutoConfiguration.java

```diff
@@ -60,17 +66,33 @@ public FailureHandler failureHandler() {
     @Bean
     @DependsOn({BEAN_NAME_SPRING_APPLICATION_CONTEXT_PROVIDER, BEAN_NAME_FAILURE_HANDLER})
     @ConditionalOnMissingBean(GlobalTransactionScanner.class)
-    public GlobalTransactionScanner globalTransactionScanner(SeataProperties seataProperties, FailureHandler failureHandler) {
+    public GlobalTransactionScanner globalTransactionScanner(
+            SeataProperties seataProperties, FailureHandler failureHandler,
```

Comment:
SeataProperties is not supposed to be used here.It's only use for spring-boot auto-configurations.